src/js/components/TaskDetailComponent.jsx

```diff
@@ -38,6 +38,29 @@ var TaskDetailComponent = React.createClass({
     );
   },
 
+  getTaskEndpoints: function () {
+    var props = this.props;
+    var task = props.task;
+    if (!task.ports) {
+      return "None";
+    }
+
+    let endpoints = task.ports.map((port) => {
+      let endpoint = `${task.host}:${port}`;
+      return (
+        <li key={endpoint} className="overflow-ellipsis">
+          <a href={endpoint} target="_blank">{endpoint}</a>
```

Comment:
No need for a list here, you can do it like the enums in AppVersionComponent.jsx.

```javascript
return task.ports.map((port) => {
  let endpoint = `${task.host}:${port}`;
  return (
    <dd key={"endpoint" + port}>
      <a href={endpoint} target="_blank">{endpoint}</a>
    </dd>
  );
});

```
Maybe this solves the text-align, too? Otherwise I am fine with it anyway.